components/UploadedItems.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+var items = 50;
+var finalURL = `https://archive.org/advancedsearch.php?q=bub.wikimedia+&fl%5B%5D=identifier&fl%5B%5D=publicdate&fl%5B%5D=title&sort%5B%5D=publicdate+desc&sort%5B%5D=&sort%5B%5D=&rows=${items}&page=1&output=json`;
```

Comment:
Make a get request to https://archive.org/advancedsearch.php?q=bub.wikimedia+&rows=0&output=json . Current url contains lot of query params that we do not require and the response has the info about uploaded books (which we don't need).

Reply:
OK, I'll change that